llvm/test/CodeGen/AMDGPU/ipra-regmask.ll

```diff
@@ -1,5 +1,10 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
 ; RUN: llc -mtriple=amdgcn-amd-amdhsa -enable-ipra -print-regusage -o /dev/null 2>&1 < %s | FileCheck %s
+
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -stop-after=irtranslator -o - %s \
```

Comment:
`irtranslator` is just for global isel but it works here to print just the MIR.

Reply:
Don't do this, this will just manifest in strange failures in the future. This doesn't make sense as a stopping point for a register allocation related test

Reply:
I wanted to test by using the printer pass, is `-stop-after=prologepilog` correct instead?

Reply:
Adding `-enable-new-pm` will work if I can insert the printer in the normal pipeline.

The `-print-regusage` options works by printing in the `doFinalization` method of the legacy pass. I can't find an equivalent mechanism in NPM. One hack is to use the destructor of the new pass but that requires a guard to prevent printing twice (analysis pass objects are destroyed twice)

Reply:
It seems to just be a general analysis print. Can we get rid of the flag and use a general mechanism?

Reply:
I'll do that